tests/utils.py

```diff
@@ -55,10 +68,26 @@ def venv_python(venv_dir: Path) -> Path:
     return venv_dir / "bin" / "python"
 
 
+# ====================================================================
+# Parsing the requirements file
+# ====================================================================
+
+
 @cache
+def parse_versions_from_requirements() -> dict[str, Requirement]:
+    """Return the requested version for each package in the requirements file."""
+    reqs: dict[str, Requirement] = {}
+    for line in open("requirements-tests.txt"):
+        line = line.split("#")[0].strip()
+        if not line:
+            continue
+        req = Requirement(line)
+        reqs[req.name] = req
+    return reqs
```

Comment:
Oh, actually, looks like we already have this function implemented here: https://github.com/python/typeshed/blob/e960f2ff77ac325c5174b098941b1534a8983e09/tests/check_consistent.py#L138-L142

We should just move that implementation here and import it from here in `check_consistent.py`